src/components/content/content.html

```diff
@@ -4,5 +4,9 @@
             <i class="paperbits-icon paperbits-send"></i>
             Publish website
         </button>
+        <button class="btn" data-bind="click: unpublish">
+            <i class="paperbits-icon paperbits-simple-remove"></i>
+            Unpublish website
```

Comment:
maybe **Reset website/content** or **Website/Content reset**? Ask Mike what is better.